README.md

```diff
@@ -113,7 +122,9 @@ end
 Your answer:
 ```text
 
-Replace this with your answer
+'describe' is used when referring to either a Class or a method. 'context' is used within a method to specify what should happen in conditionals.
+
```

Comment:
Yes, that's correct! Functionally, they are the same but used in different situations